### PR TITLE
Loginendpoint

### PR DESCRIPTION
**Pull request by x00145570**
Pull this

### CODE REVIEW

File: POSAPI/Migrations/20230530215247_SystemUser.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                columns: table => new
+                {
+                    ID = table.Column<string>(type: "varchar(255)", nullable: false)
+                        .Annotation("MySql:CharSet", "utf8mb4"),
+                    DisplayName = table.Column<string>(type: "longtext", nullable: true)
+                        .Annotation("MySql:CharSet", "utf8mb4"),
+                    Username = table.Column<string>(type: "longtext", nullable: true)
+                        .Annotation("MySql:CharSet", "utf8mb4"),
+                    Password = table.Column<string>(type: "longtext", nullable: true)
+                        .Annotation("MySql:CharSet", "utf8mb4")
```
Comment (ppivanov) [inline POSAPI/Migrations/20230530215247_SystemUser.cs:25]: can probably be varchar(255)

File: POSAPI/Migrations/ModelModelSnapshot.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                        .HasColumnType("varchar(255)");
+
+                    b.Property<string>("DisplayName")
+                        .HasColumnType("longtext");
+
+                    b.Property<string>("Password")
+                        .HasColumnType("longtext");
+
+                    b.Property<string>("Username")
+                        .HasColumnType("longtext");
```
Comment (ppivanov) [inline POSAPI/Migrations/ModelModelSnapshot.cs:33]: same

File: POSAPI/Migrations/20230530215247_SystemUser.Designer.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                        .HasColumnType("varchar(255)");
+
+                    b.Property<string>("DisplayName")
+                        .HasColumnType("longtext");
+
+                    b.Property<string>("Password")
+                        .HasColumnType("longtext");
+
+                    b.Property<string>("Username")
+                        .HasColumnType("longtext");
```
Comment (ppivanov) [inline POSAPI/Migrations/20230530215247_SystemUser.Designer.cs:35]: same

File: POSAPI/src/Controllers/UserController.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+using Microsoft.AspNetCore.Mvc;
+using POSAPI.src;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+
+namespace POSAPI.Controllers
+{
+    [Route("api/services")]
```
Comment (ppivanov) [inline POSAPI/src/Controllers/UserController.cs:13]: wanna keep this as services?

Comment (x00145570) [reply POSAPI/src/Controllers/UserController.cs:13]: No

Comment (x00145570) [reply POSAPI/src/Controllers/UserController.cs:13]: would pos be okay?

**Review by ppivanov — APPROVED** [review]

**Pull request merged**